pkg/apis/core/v1/defaults.go

```diff
@@ -376,8 +376,8 @@ func SetDefaults_RBDVolumeSource(obj *v1.RBDVolumeSource) {
 }
 
 func SetDefaults_RBDPersistentVolumeSource(obj *v1.RBDPersistentVolumeSource) {
-	if obj.RBDPool == "" {
-		obj.RBDPool = "rbd"
+	if obj.Pool == "" {
+		obj.Pool = "rbd"
```

Comment:
@rootfs @thockin i believe there is no reasonable default we can assume for a rbd pool. Is there a good reason to assume that if none is specified we should assume `rbd` ? Is that specific to GCE or some such environment. IMO we should error out if pool is empty

Reply:
agree, this is actually what's in luminous release now: no more default rbd pool.

Reply:
Well, this PR is not about that - I have no real say how RBD logic behaves.  That said, this is part of the API, so changing it would be a breaking change.

Reply:
that's fair, let it for later.

Reply:
@rootfs should i create a issue to remove this default ?